src/wallet/rpcwallet.cpp

```diff
@@ -1285,7 +1285,8 @@ UniValue addwitnessaddress(const JSONRPCRequest& request)
     if (request.fHelp || request.params.size() < 1 || request.params.size() > 2)
     {
         std::string msg = "addwitnessaddress \"address\" ( p2sh )\n"
-            "\nAdd a witness address for a script (with pubkey or redeemscript known). Requires a new wallet backup.\n"
+            "\nDEPRECATED: use option -addresstype=[bech32|p2sh-segwit] instead.\n"
```

Comment:
Wouldn't it be better to suggest using the RPC parameters, rather than changing the config file and restarting?

Reply:
Yeah `getnewaddress` with the `address_type` parameter more closely resembles what this RPC was used for IMO

Reply:
best to document both, I'll mention that option as well.